src/components/ReadMore/index.js

```diff
@@ -49,9 +50,18 @@ const ReadMore = ({text, onPress}) => {
       ) : null}
       {!isFinishSetLayout ? (
         <TouchableOpacity testID="finishLayout" onPress={onPress}>
-          <Text style={styles.text} testID="notFinishLayout" onTextLayout={handleLayoutText}>
+          {/* <Text style={styles.text} testID="notFinishLayout" onTextLayout={handleLayoutText}>
             {text}{' '}
-          </Text>
+          </Text> */}
+          <TextWithEmoji
+            testId="notFinishLayout"
+            text={text}
+            textStyle={{
+              marginBottom: 0,
+              fontSize: normalizeFontSize(14),
+              lineHeight: undefined
+            }}
+          />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Replacing the `<Text>` component with `<TextWithEmoji>` is fine if the `TextWithEmoji` component provides additional functionality needed in this context. However, I noticed that you're setting `lineHeight: undefined` in the `textStyle` prop. If the `TextWithEmoji` component uses `lineHeight` in its styles, this could potentially override it and cause unexpected behavior. Make sure this is the intended effect.
```diff
- lineHeight: undefined
+ lineHeight: normalizeFontSize(16) // or any other value that suits your design
```
Also, consider moving the inline style to the `styles` object at the top of the file for better readability and maintainability.


<!-- This is an auto-generated comment by OSS CodeRabbit -->